src/Client.php

```diff
@@ -131,6 +131,9 @@ public function __call(string $functionName, array $args)
 
         $bag = new HeadersOutgoing($headers);
         $context = SerializationContext::create()->setAttribute('headers_outgoing', $bag);
+        $context->setAttribute('soapAction', $soapOperation['action']);
```

Comment:
this looks reasonable, why not passing the while `$soapOperation` ? i think that in there there are all the info somenone might need, without having to inject them one by one

Reply:
I wanted to be explicit about it, as I am not sure which other properties are always there, but for this one I know. not sure if I should add all, but if you think I should, fine by me.